core/trino-main/src/main/java/io/trino/sql/analyzer/Analysis.java

```diff
@@ -503,6 +509,26 @@ public void setOrderByWindowFunctions(OrderBy node, List<FunctionCall> functions
         return orderByWindowFunctions.get(NodeRef.of(query));
     }
 
+    public void setWindowMeasures(QuerySpecification node, List<Measure> measures)
```

Comment:
FunctionCalls and measures require both different actions in the analyzer, and different planning.

To some extent, they could be handled together, but this is currently inconvenient because they have different api wrt access to `Window`.
I'll keep them separate for now, and once we refactor `FunctionCall`, we'll be able to decide where we can merge them, and where we'd rather keep them distinct.